js/components/analysisExecution/analysis-execution-list.js

```diff
@@ -319,7 +321,7 @@ define([
     }
 
     goToResults(executionId) {
-      CommonUtils.routeTo(`${this.resultsPathPrefix}${this.analysisId()}/results/${executionId}`);
+      CommonUtils.routeTo(`${this.resultsPathPrefix}${this.characterizationId()}/results/${executionId}`);
```

Comment:
?

Reply:
this changed is based on the assumption that it is related to this: https://github.com/uc-cdis/Atlas/pull/78/files#diff-a4e940137831abe34af9ada8e2e50b8928d570169901efcea71a3ad6237b8b12R46-R47

Reply:
AND this: https://github.com/uc-cdis/Atlas/blob/63a891a3133e256b278266f2abae6bb63edcc060/js/pages/characterizations/components/characterizations/characterization-view-edit/characterization-exec-wrapper.html#L3 
and 
https://github.com/uc-cdis/Atlas/blob/63a891a3133e256b278266f2abae6bb63edcc060/js/pages/characterizations/components/characterizations/characterization-view-edit/characterization-exec-wrapper.js#L42